index.rst

```diff
@@ -98,6 +98,17 @@ Audio
 
     <div style='clear:both'></div>
 
+Named Tensor
+----------------------
+
+.. customgalleryitem::
+   :figure: /_static/img/named_tensor.png
```

Comment:
What is the image?

Reply:
<img width="491" alt="named_tensor" src="https://user-images.githubusercontent.com/13398637/66428828-a0633580-e9cb-11e9-942d-3aefa1637b5c.png">

Reply:
What _should_ the logo be? I'm at a loss here